GoogleForCreators/web-stories-wp
# Color picker redesign

## Context



## Summary
Redesigns the Color Picker based on [new design](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=1965%3A326097).


## Relevant Technical Choices



## To-do

- [x] Deleting a gradient stop (needs specs from UX) -- via keyboard only
- ~Fix jumpiness when dragging gradient stops.~ Seems to be happening locally only and on `main`, too. If this appears elsewhere then we should create a bug issue for it, not related to redesign. Added an issue #6426 



## User-facing changes
| Before  | After |
| ------------- | ------------- |
| <img width="259" alt="Screenshot 2021-02-18 at 16 30 33" src="https://user-images.githubusercontent.com/3294597/108371703-519fef80-71fe-11eb-898e-f10249dee1e3.png"> | <img width="254" alt="Screenshot 2021-02-18 at 16 28 23" src="https://user-images.githubusercontent.com/3294597/108371455-0dacea80-71fe-11eb-975c-fd8ca3ff48c4.png"> |
| <img width="256" alt="Screenshot 2021-02-18 at 16 30 45" src="https://user-images.githubusercontent.com/3294597/108371762-5e244800-71fe-11eb-9775-aace1f33233d.png"> | <img width="259" alt="Screenshot 2021-02-18 at 16 28 56" src="https://user-images.githubusercontent.com/3294597/108371505-1bfb0680-71fe-11eb-9ba3-e61c0bed3217.png"> |


## Testing Instructions
1. Add a shape and open the color picker
2. Verify that Solid color style looks similar to Figma
3. Verify that Linear and Radial also look similar to Figma

Note: there is a bug related to moving stops by dragging -- there's jumpiness and the stops aren't placed properly. This is an unrelated bug and will be addressed in a separate PR.

Note 2: The dropdown in the saved colors area will be redesigned separately.


### QA
See the testing instructions in the previous point.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #6208
